g3doc/Tutorials/Toy/Ch-2.md

```diff
@@ -25,7 +25,7 @@ terminology) or types.
 
 ## MLIR Module, Functions, Blocks, and Operations
 
-[Language reference](LangRef.md#operations)
+[Language reference](https://github.com/tensorflow/mlir/blob/master/g3doc/LangRef.md#operations)
```

Comment:
Can you use a relative link? `../../LangRef.md` ?

Reply:
Fixed